batchDivision/batch_division.go

```diff
@@ -0,0 +1,80 @@
+package batchDivision
+
+import (
+	"github.com/codeuniversity/al-proto"
+	"math"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+func createBatches(cells []*proto.Cell, batchSize uint) map[string][]*proto.Cell {
+	dict := make(map[string][]*proto.Cell)
+
+	for i := 0; i < len(cells); i++ {
```

Comment:
you can just use the `range` keyword:
```golang
for _, cell := range cells {
...
}
```
then you don't need to use `cells[i]` everywhere :)

Reply:
didn't know that existed in go :smiley: